### PR TITLE
Added sort option for po.compile

### PR DESCRIPTION
**Pull request by AlexMost**
This PR adds sorting to the resulting .po(.pot) file by msgid. 

Sorting can be handy for avoiding merge conflicts in .po(.pot) files. 
Currently working on this tool - https://c-3po.js.org/ and using gettext-parser for creating .po(.pot) files and have this feature request to add sorting option.

You can check [this issue](https://github.com/c-3po-org/c-3po/issues/11) for other details.

### CODE REVIEW

**Comment by smhg** [comment]
(I have no idea why this repo still doesn't send me notifications - maybe because it was moved to me)

Thank you for this feature! It looks good to me.
Can you update with the changes in master though?
And maybe move the compare function outside of `compile`, to the top of the file.

Thanks again!

**Comment by smhg** [comment]
Just to be clear: use the `options` in the README instead of `compileOptions` when you merge.

**Comment by smhg** [comment]
@AlexMost any chance you'd be able to update the PR? Would be great to create a new release with this feature.

**Comment by AlexMost** [comment]
@smhg oh, sorry for the delay, I have completely forgotten about this PR, will try to update it ASAP.

**Comment by AlexMost** [comment]
@smhg master is merged, looks like the tests are green.

File: .gitignore [hunk 1]
```diff
@@ -14,4 +14,6 @@ results
 npm-debug.log
 node_modules
 .DS_Store
+.idea
 package-lock.json
+.idea
```
Comment (smhg) [inline .gitignore]: If you can remove one of these .idea ignores, we are good to go. Thx!

Comment (AlexMost) [reply .gitignore]: ah, sure. Removed.

**Comment by smhg** [comment]
Thanks a lot!

**Pull request merged**